src/components/AppHeading.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <h2 class="font-title font-semibold text-xl text-body-80"><slot /></h2>
+  <h2 class="font-title font-semibold text-xl text-body-40"><slot /></h2>
```

Comment:
@rlafuente `AppHeading` are using 40% opacity text too

Reply:
Ah I see from your comment you meant you want to introduce a 60% opacity for these components?

Reply:
yes that's it!